### PR TITLE
Rename repo to `vision-zero`

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/19146

## Testing

Check out the footer links in the [VZV deploy preview](https://deploy-preview-1566--atd-vzv-staging.netlify.app/). Make sure the **Code** link works.

---

#### Ship list

- [ ] Check migrations for any conflicts with latest migrations in `main` branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: database/README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ For example, consider the `lookups.injry_sev` table, which includes a custom val
 | 99  | KILLED (NON-ATD)         | vz     |
 ```
 
-The original migration for this table is [here](https://github.com/cityofaustin/atd-vz-data/blob/e56e3c6bc654a21f667142ce53232bad44cff7e5/atd-vzd/migrations/default/1715960018005_lookup_table_seeds/up.sql#L11054-L11056).
+The original migration for this table is [here](https://github.com/cityofaustin/vision-zero/blob/e56e3c6bc654a21f667142ce53232bad44cff7e5/database/migrations/default/1715960018005_lookup_table_seeds/up.sql#L11054-L11056).
```
[the remark this answers is not in the document]
Comment (chiaberry) [reply database/README.md:124]: yeah the part that says database should stay atd-vzd

[the remark this answers is not in the document]
Comment (chiaberry) [reply database/README.md:124]: https://github.com/cityofaustin/vision-zero/blob/e56e3c6bc654a21f667142ce53232bad44cff7e5/atd-vzd/migrations/default/1715960018005_lookup_table_seeds/up.sql#L11054-L11056)

[the remark this answers is not in the document]
Comment (chiaberry) [reply database/README.md:124]: bah, i am trying to paste the link, ill keep editing

[the remark this answers is not in the document]
Comment (johnclary) [reply database/README.md:124]: oh riiight got it thank you🙏

File: toolbox/get_lookup_table_changes/README.md
Comment (chiaberry) [inline toolbox/get_lookup_table_changes/README.md:1]: I found a place on L56 that i updated from atd-vzd to database in my PR but never pushed up :( 
`3. To test the migrations, navigate to `./atd-vzd` and create a new migration with the Hasura console.`

**Review by chiaberry — APPROVED** [review]
📚 and the link to the code works ❇️

**Pull request merged**